Paskab012/cartix-web1
# Refactor reduce to return 0 when item is not present

#### Hello Reviewer ✋🏽 
## Description of changes made ♻️
Implements wrapper function to Array.reduce to return 0 when item is not in obj

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/Paskab012/cartix-web1.git)
- Check out this branch by `integration`
- Run `yarn or npm install to install the project dependencies
- Run `npm run start:cartix` to start the development server

#### Checklist


#### Screenshots

|        BEFORE                  |                     AFTER       |
| :----------------------------: | :-----------------------------: |
|   data.reduce((accum, item) => accum + item[""])        | reduce(data, key) => ...|
|                                | screenshot                      |
